### PR TITLE
Xaiki/css flags

### PR DESCRIPTION
**Pull request by xaiki**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,27 +1,54 @@
...
-    }
+  "name": "butter",
+  "dependencies": {
+    "marionette": "^2.4.1",
+    "bootstrap": "~3.3.7",
+    "video.js": "4.11.4",
+    "videojs-youtube": "1.2.10",
+    "mousetrap": "~1.6.0",
+    "font-awesome": "^4.6.3",
+    "flag-icon-css": "^2.7.0"
```
Comment (vankasteelj) [inline bower.json]: why is that in both bower & npm?

Comment (xaiki) [inline bower.json]: that's a mistake, clearly, i'd like to have it in npm, but i'm not sure how to include it...

**Comment by xaiki** [comment]
i haven't removed the old ones because some i still use, and we need to go through all the languages to find the ones that we'd want to keep (chinese has a split china/tw flag for instance).

this PR started because now that we get subs, a lot of flags were empty.

File: bower.json (hunk 1 above)
Comment (vankasteelj) [inline bower.json]: I really don't know^^ worth some searching though.

File: src/app/language.js [hunk 2]
```diff
@@ -158,7 +176,8 @@ App.Localization.langcodes = {
     },
     'ca': {
         name: 'Catalan',
-        nativeName: 'Català'
+        nativeName: 'Català',
+        flag: 'catalan'
```
Comment (vankasteelj) [inline src/app/language.js]: why not flag:'ca' or 'es-ca'? it's a little weird to have some flags using a 2 letter langcode, other full name. Not really important, but might worth considering

Comment (xaiki) [inline src/app/language.js]: so 2 letter codes come from the flag library, the full names are the ones we provide. for arabic it was confusing as ar = argentina ? es-ar ? ar... some would say that catalan has nothing to do with spanish... that said we have catalan and occitan, and i'm pretty sure fr-oc is wrong.

Comment (wallydz) [inline src/app/language.js]: AR is the ISO code for arabic lang
.ar is the TLD for argentina :p

**Comment by vankasteelj** [comment]
it's ugly as hell

**Comment by xaiki** [comment]
closing this, on twitter people asked for flags, we'll wait to see what @abad says with design.

**Pull request closed without merging**